packages/kit/src/runtime/client/router.js

```diff
@@ -169,6 +197,14 @@ export class Router {
 		addEventListener('popstate', (event) => {
 			if (event.state && this.enabled) {
 				const url = new URL(location.href);
+
+				const canNavigate = triggerNavigationIntentListener(url);
+				if (!canNavigate) {
+					//"disabling" the back/forward button click by pushing the last known location
```

Comment:
```suggestion
					// "disabling" the back/forward button click by pushing the last known location
```

Reply:
Do I need to do any changes here or just the spacing?